WordPress/src/main/java/org/wordpress/android/ui/reader/views/ReaderExpandableTagsView.kt

```diff
@@ -160,4 +166,61 @@ class ReaderExpandableTagsView @JvmOverloads constructor(
             }
         })
     }
+
+    private sealed class ChipStyle(val name: String) {
+        abstract val chipLayoutRes: Int
```

Comment:
Nitpick: would it make sense to add `@get:LayoutRes` to the `chipLayoutRes` abstract property to enforce the contract with child classes so they provide a real layout `Int`?

<img width="525" alt="image" src="https://github.com/wordpress-mobile/WordPress-Android/assets/14964993/daf62291-45a4-4292-be61-2846107aba5b">

Reply:
Thanks for this tip, I tried adding the `LayoutRes` but missed the `get:` part needed to make it work for the abstract field.